src/lazy/binary/raw/v1_1/struct.rs

```diff
@@ -163,10 +181,13 @@ impl<'top> RawBinaryStructIterator_1_1<'top> {
         let (flex_sym, after) = buffer.read_flex_sym()?;
         let (sym, after) = match flex_sym.value() {
             FlexSymValue::SymbolRef(sym_ref) => (sym_ref, after),
-            FlexSymValue::Opcode(_opcode) => todo!(),
+            FlexSymValue::Opcode(Opcode{ opcode_type: OpcodeType::DelimitedContainerClose, ..}) => {
+                return Ok(None)
+            }
```

Comment:
Slightly more concise:
```rust
FlexSymValue::Opcode(o) if o.opcode_type == `DelimitedContainerClose` => {...}
```

Or it might be nice to have an `Opcode::is_delimited_end()` helper method?